### ISSUE TITLE
i cant get to the site localhost:3

### ISSUE THREAD

**Issue by omeglere** [ash47/OmegleMiddleMan#72]

**Comment by ash47** [comment]
This app uses port 3000, not port 3. Please also ensure you run the app.

**Comment by omeglere** [comment]
i runned the app but it says
 This site can’t be reachedlocalhost refused to connect.
Try:

Checking the connection
Checking the proxy and the firewall
ERR_CONNECTION_REFUSED

**Comment by omeglere** [comment]
also the spymode doesnt work and i cant see the videos of the other ppls webcams

**Comment by ash47** [comment]
Yeah, that feature in this app is broken, please look at the 30 other
issues that have been opened about that -- I'm focused on other projects
and not working on this right now
